motion/thruster_interface_auv_cpp/src/thruster_interface_auv_driver.cpp

```diff
@@ -0,0 +1,138 @@
+#include "thruster_interface_auv_driver.hpp"
```

Comment:
Include paths for the header files of this project should be relative to the include folder